tests/test_wave_06.py

```diff
@@ -129,3 +129,46 @@ def test_swap_best_by_category_no_other_match():
     assert item_a in tai.inventory
     assert item_b in tai.inventory
     assert item_c in tai.inventory
+
+
+
+def test_get_newest():
```

Comment:
You should also test the `get_newest()` when one or another list is empty.

You should also write a test for what happens when two items are the same age.